x/concentrated-liquidity/position.go

```diff
@@ -62,6 +63,37 @@ func (k Keeper) initOrUpdatePosition(
 
 	// TODO: consider deleting position if liquidity becomes zero
 
+	// Create records for relevant uptime accumulators here.
+	uptimeAccumulators, err := k.getUptimeAccumulators(ctx, poolId)
+	if err != nil {
+		return err
+	}
+
+	for uptimeId, uptime := range types.SupportedUptimes {
```

Comment:
nit: I was confused for a second why uptimes have ids
```suggestion
	for uptimeIndex, uptime := range types.SupportedUptimes {
```

Reply:
Updating